src/features/newTab/default/settings/index.tsx

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import { StyledSettingsMenu, StyledSettingsRow, StyledSettingsText, StyledSettingsTitle } from './style'
+
+import { Toggle } from '../../../shields/toggle'
```

Comment:
We can't add any more cross references between components, especially for NTP. Please move this Toggle to root brave-ui level, or local to NTP @cezaraugusto

Reply:
let's copy this component to NTP for now so @imptrx is not blocked. we can make it a standard once we port NTP fully to brave-core